src/orion/executor/single_backend.py

```diff
@@ -16,14 +16,23 @@
     Future,
 )
 
+class _None:
```

Comment:
I think NOT_SET would be less confusing. You can also instantiate a global object instead of defining a new class, like here: https://github.com/Epistimio/orion/blob/develop/src/orion/core/io/config.py#L21.

Reply:
done

Reply:
Thanks!